src/Services/PatientService.php

```diff
@@ -39,15 +39,6 @@ public function __construct()
         parent::__construct('patient_data');
```

Comment:
at line @19 we can skip the import of Validator as we are using PatientValidator 🤔